src/ui/public/management/index_pattern_creation/index_pattern_creation_config.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { i18n } from '@kbn/i18n';
+
+const indexPatternTypeName = i18n.translate('kbn.management.editIndexPattern.createIndex.defaultTypeName',
```

Comment:
Could you show me an example of how to wrap it in a function? We need to use that pattern here because the constructor uses these values as defaults for its arguments.

I will also update the docs to reflect your comments (https://github.com/elastic/kibana/tree/master/packages/kbn-i18n#vanilla-js).

Reply:
This limitation will be fixed by the PR #23902 for the **client** code. That is that [this](https://github.com/elastic/kibana/pull/21117/files#diff-a4d19e14e3f7c55ce657093abc571f9dR21) translations will work after the PR is merged.

For **server** side code we should wrap constants like in tutorials: https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/common/tutorials/logstash_instructions.js#L22